### PR TITLE
Unreg user nav

### PR DESCRIPTION
**Pull request by philjdelong**
User can visit the home page and see links for logo(home), login, register, artists, locations in the nav bar

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -41,6 +41,11 @@ group :development, :test do
   gem 'pry'
   gem 'capybara'
```
Comment (darren2802) [inline Gemfile]: I think capybara, rspec-rails, shoulda-matchers and simplecov should be in the test block and not needed in development?

Comment (wrenisit) [reply Gemfile]: That sounds good.

Comment (wrenisit) [reply Gemfile]: Same thing with Launchy.

**Comment by LainMcGrath** [comment]
build is failing because travis.yml file needs to look like this 

```
services:
    - postgresql
language: ruby
rbenv:
  - 2.4.1
script:
    - bundle exec rails db:reset db:setup db:migrate RAILS_ENV=test
```
Once file is updated, build should pass. maybe?

**Pull request merged**